src/app/components/engangsstonad/steps/Engangsstonad.step2.js

```diff
@@ -1,25 +1,41 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
 
 import { Normaltekst, Element } from 'nav-frontend-typografi';
 import { ToggleGruppe, ToggleKnapp } from 'nav-frontend-skjema';
 
+import {
+    toggleChildBorn,
+    enableNextButton,
+    disableNextButton,
+    toggleNoOfChildren,
+    setTerminDato,
+    setBekreftetTermindato
+} from '../../../redux/ducks/Engangsstonad.duck';
+
 import DialogBox from '../../shared/dialog-box/DialogBox';
 import DateInput from '../../shared/date-input/DateInput';
 import AttchmentList from '../../shared/attachment-list/AttachmentList';
 import AttchmentButton from '../../shared/attachment-button/AttachmentButton';
 
-class Step2 extends Component {
-    constructor(props) {
-        super(props);
-        this.toggleBirth = this.toggleBirth.bind(this);
+export class Step2 extends Component {
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.childBorn === 'childBorn' ||
```

Comment:
Ville ikke håndteringen av childBorn blitt enklere (og koden litt tydeligere) om det var en boolean istedetfor en string som settes til 'childBorn' eller 'childNotBorn'? Eller er det en grunn for at det er hensiktsmessig å ha som string?

Reply:
Hoppet over til string siden valuen til change eventen er valuen til radio optionet som er string. Kan konverteres til boolean for å gjøre det koseligere.